src/css/intlTelInput.scss

```diff
@@ -229,6 +229,11 @@ $flagsImageExtension: "png" !default;
     height: 100%;
     opacity: 0;
   }
+
+  // if container option is set, increase z-index to prevent display issues
+  &.iti-container {
+    z-index: 1060;
```

Comment:
The default bootstrap modal z-index is 1050. 1060 allows the menu to appear above a modal and should cover most use cases and can always be customized if need be. A z-index should be set by default, though, to make this feature usable out of the box, but I'm not opposed to changing it.